expyre/func.py

```diff
@@ -429,8 +434,24 @@ def sync_results(self, sync_all=True, force_sync=False, verbose=False):
         else:
             status = 'ongoing'
 
-        jobs_to_sync = list(config.db.jobs(system=self.system_name, id=job_id, status=status))
+        jobs_to_sync = list(config.db.jobs(system=self.system_name, id=re.escape(job_id), status=status))
```

Comment:
`re.escape` fails if `job_id` is `None`. Could put the escape in line 429?

Reply:
Yes - this is one of the things I fixed, but forgot to push.  I like your fix better, so I changed it.  Commit coming in a few minutes.